scripts/download.js

```diff
@@ -0,0 +1,237 @@
+  async function getFileContentHns(hnsLink) {
+    await client
+      .getFileContentHns(hnsLink)
+      .then((res) => {
+        console.log("\n\n\n9. use getFileContentHns to get the hns fileContent from a hnsLink.");
+        console.log("data: " + res.data);
+        console.log("contentType: " + res.contentType);
+        console.log("portalUrl: " + res.portalUrl);
+        console.log("skylink: " + res.skylink);
+      })
+      .catch((err) => {
+        console.log("\n9. Get Error: ", err);
+      });
+  }
+
+  //10. use getFileContentBinaryHns to get the hnsFileContentBinary from a hnsLink.
+  async function getFileContentBinaryHns(hnsLink) {
+    await client
+      .getFileContentBinaryHns(hnsLink)
+      .then((res) => {
+        console.log("\n\n\n10. use getFileContentBinaryHns to get the hnsFileContentBinary from a hnsLink.");
+        console.log("binaryData: " + res.data);
+        console.log("contentType: " + res.contentType);
+        console.log("portalUrl: " + res.portalUrl);
+        console.log("skylink: " + res.skylink);
+      })
+      .catch((err) => {
+        console.log("\n10. Get Error: ", err);
+      });
+  }
+
+  //11. use openFile to open a skylink in browser.
+  async function openFile(skylink) {
+    await client
+      .openFile(skylink)
+      .then((res) => {
+        console.log("\n\n\n11. use openFile to open a skylink in browser.");
+        console.log("skylinkUrl: " + res);
+      })
+      .catch((err) => {
+        console.log("\n11. Get Error: ", err);
+      });
+  }
+
+  //12. use openFileHns to open a hnsLink in browser.
+  async function openFileHns(hnsLink) {
+    await client
+      .openFileHns(hnsLink)
+      .then((res) => {
+        console.log("\n\n\n12. use openFileHns to open a hnsLink in browser.");
+        console.log("hnsUrl: " + res);
+      })
+      .catch((err) => {
+        console.log("\n12. Get Error: ", err);
+      });
+  }
+
+  //13. use resolveHns to get from a hnsLink the data.
+  async function resolveHns(hnsLink) {
+    await client
+      .resolveHns(hnsLink)
+      .then((res) => {
+        console.log("\n\n\n13. use resolveHns to get from a hnsLink the data.");
+        console.log("data: " + JSON.stringify(res.data));
+        console.log("skylink: " + res.skylink);
+      })
+      .catch((err) => {
+        console.log("\n13. Get Error: ", err);
+      });
+  }
+
+  async function main() {
+    await downloadFile(defaultDownloadPath + "sia.pdf", usedSkylink);
+    await downloadFileHns(defaultDownloadPath + usedHnsLink + ".html", usedHnsLink);
+    await getSkylinkUrl(usedSkylink);
+    await getHnsUrl(usedHnsLink);
+    await getHnsresUrl(usedHnsLink);
+    await getMetadata(usedSkylink);
+    await getFileContent(usedSkylink);
+    await getFileContentBinary(usedSkylink);
+    await getFileContentHns(usedHnsLink);
+    await getFileContentBinaryHns(usedHnsLink);
+    await openFile(usedSkylink);
+    await openFileHns(usedHnsLink);
```

Comment:
I see that these actually work and open the links in the default browser. Interesting! I'm not convinced it's something we want yet, I would remove it for now and we might add it back in later. It seems to work well though!

Reply:
hello m-cat,

I take it out.